extensions/wikia/FacebookClient/FacebookClientHooks.class.php

```diff
@@ -154,17 +154,13 @@ public static function onSkinTemplatePageBeforeUserMsg( &$html ) {
 	}
 
 	/**
+	 * SUS-905: Load FBTags JS in a single HTTP request with the other groups
+	 * @param OutputPage $out
 	 * @param Skin $skin
-	 * @param string $text
-	 *
-	 * @return bool
-	 * @throws WikiaException
+	 * @return bool true to continue hook processing
 	 */
-	public static function onSkinAfterBottomScripts( $skin, &$text ) {
-
-		$script = AssetsManager::getInstance()->getURL( 'facebook_client_fbtags_js' );
-		$text .= Html::linkedScript( $script[0] );
-
+	public static function onBeforePageDisplay( OutputPage $out, Skin $skin ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameters "$out, $skin". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)